experiment/run_experiment.py

```diff
@@ -88,28 +88,27 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
     for param in required_params:
         if param not in config:
             valid = False
-            logs.error('Config does not contain "%s".', param)
+            logs.error(f'Config does not contain "{str(param)}".')
```

Comment:
This pattern is going to beak presubmit.
The standard pattern we adhere to is to use % format strings for logs, not f-strings.